bin/fasta-to-phylip.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import tempfile
+
+from dark.fasta import FastaReads
+from dark.fasta_ss import SSFastaReads
+from dark.fastq import FastqReads
+
+
+if __name__ == '__main__':
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description=('Given FASTA on stdin write the sequence ids and lengths '
+                     'to stdout.'))
+
+    parser.add_argument(
+        '--readClass', default='fasta', choices=('fasta', 'fastq', 'fasta-ss'),
```

Comment:
BTW, this is a classic development pattern. Some people argue that it's better to fix stuff partially, while others don't want to do that because they don't believe the rest of the cleaning up will ever be done. I'm in the latter camp. If it's important enough to change, it can have its own ticket and be done properly everywhere, etc.  Some people might expand the scope of their pr to make unrelated (to the original ticket) changes elsewhere, but that can annoy reviewers who are trying to evaluate on change.